includes/core-changes/msbuild/3.0/resource-manifest-name.md

```diff
@@ -0,0 +1,89 @@
+### Resource file names
+
+Starting in .NET Core 3.0, the generated resource manifest file name has changed.
+
+#### Version introduced
+
+3.0
+
+#### Change description
+
+Prior to .NET Core 3.0, when [DependentUpon](/visualstudio/msbuild/common-msbuild-project-items#compile) metadata was set for a resource file in the MSBuild project file, the generated manifest name was *Namespace.Classname.resources*. When [DependentUpon](/visualstudio/msbuild/common-msbuild-project-items#compile) was not set, the generated manifest name was Namespace.Classname.FolderPathRelativeToRoot.Culture.resources.
+
+Starting in .NET Core 3.0, when a *.resx* file is associated with a type, for example, in Windows Forms apps, it's assumed that the resource is named *Namespace.Classname\[.Culture].resources*. From this convention, the rules around how the root namespace and folders impact the chosen name mirror how Visual Studio picks a namespace and name for a new class: *RootNamespace.FoldersSeparatedByDots.FileName.(resx|resources|cs)*.
+
+##### Rules that determine resource file names
+
+If you leave all the defaults when you add files and resources in Visual Studio, then when *Type.cs* is colocated with *Type.resx*, the generated manifest name is *Namespace.Classname.resources*.
+
+If you don't use the default settings, there are more considerations as to how the manifest name is generated:
+
+- If the `LogicalName` attribute on the `EmbeddedResource` element is set, that value is used as the resource file name.
+
+  Examples:
+
+  `<EmbeddedResource Include="X.resx" LogicalName="SomeName.resources" />`
```

Comment:
These two lines would look better in a fenced code block, or with a blank line between them.